tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -41,18 +41,18 @@ bool IsRetriable(Status status) {
 
 Status CallWithRetries(const std::function<Status()>& f,
                        const initial_delay_seconds) {
-  int attempt = 0;
+  int retry = 0;
```

Comment:
Just one last comment -- suggest renaming to 'retries', since it's going to be a retry count now.